rfq/oracle.go

```diff
@@ -335,13 +321,25 @@ func (r *RpcPriceOracle) QueryBidPrice(ctx context.Context,
 		return nil, err
 	}
 
+	var subjectSpecifier oraclerpc.AssetSpecifier
+
+	switch {
+	case assetSpecifier.HasId():
+		assetID := assetSpecifier.UnwrapIdToPtr()
+		subjectSpecifier.Id = &oraclerpc.AssetSpecifier_AssetId{
+			AssetId: assetID[:],
+		}
+
+	case assetSpecifier.HasGroupPubKey():
+		groupKey := assetSpecifier.UnwrapGroupKeyToPtr()
+		subjectSpecifier.Id = &oraclerpc.AssetSpecifier_GroupKey{
+			GroupKey: groupKey.SerializeCompressed(),
+		}
+	}
```

Comment:
this could be an RPC marshal function in the RFQ package, could be called in `QueryAskPrice` also:
```
rpcMarshalAssetSpecifier(asset.Specifier) oraclerpc.AssetSpecifier
```